### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by emberborja**
I added to the readme, to describe the behavior of pac-man original game.  the terminology will change as we change the code.

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -12,4 +12,91 @@ There are no requirements at this time.
...
+
+Who is Pac-man? Why does he eat pellets? Why do the four ghosts pursue him?  Does Pac-man destroy his own universe at level 256, only to see it reborn anew?  It is possible that Pac-man himself does not know, but we can attempt to gain personal understanding through observation of his ceaseless performance of destruction and rebirth.
+
+
+PAC-MAN (the man)
+
+Pac-man is yellow and circular. His mouth opens and closes continuously.  He initially moves slightly faster than the ghosts.  He begins each level moving to the left.  He will move forward until he hits a wall.  When he hits a wall, he will face the wall and his mouth will continue to open and close, but he will not move in any direction.  Even when not moving, Pac-man will be pursued by the ghosts.  He must continue to move and to eat, or he will perish and his torment will start anew.
```
Comment (rcbrowder) [inline docs/README.md:32]: How fast is 'slightly faster' than the ghosts?

**Review by rcbrowder — APPROVED** [review]
It's a little more verbose than expected but I think it hits on everything needing explaining haha

**Comment by KarKunze** [comment]
"It is possible that Pac-man himself does not know, but we can attempt to gain personal understanding through observation of his ceaseless performance of destruction and rebirth." 
Are you sure you want to go with the Sisyphus analogy instead of PacMan as a phoenix rising from the ashes?

File: docs/README.md (hunk 1 above)
Comment (emberborja) [reply docs/README.md:32]: that is a great question that I could not find a number for.  But, I had to stop going down the rabbit hole.

**Review by TheErikWolfe — APPROVED** [review]

**Pull request merged**